Engines/Wine/Tools/Configure Wine/script.js

```diff
@@ -2,14 +2,15 @@ include("engines.wine.engine.object");
 
 /**
  * tool to configure Wine
-*/
-var toolImplementation = {
-    run: function (container) {
+ */
+class ConfigureWineTool {
+    constructor() {
+        // do nothing
+    }
+
+    run(container) {
```

Comment:
I believe `execute` to be a better name instead of `run`. @plata @qparis @ImperatorS79 what do you think?

Reply:
No preference from my side.

Reply:
Run is fine IMHO